daml-script/export/src/main/scala/com/daml/script/export/TreeUtils.scala

```diff
@@ -322,7 +325,7 @@ object TreeUtils {
           }
         }
         val (created, consumed) = treeEventCreatedConsumedCids(Kind.Exercised(exercisedEvent), tree)
-        val netCreated = (extraCreated ++ created) -- consumed
+        val netCreated = (extraCreated ++ created.map(_.cid)) -- consumed
```

Comment:
Not your fault, but always better to be explicit

```
        val netCreated = (extraCreated union created.view.map(_.cid)) -- consumed
```

or 

```
        val netCreated = (extraCreated | created.view.map(_.cid)) -- consumed
```

Reply:
in a later commit `extraCreated` and `created` become `Seq`s (and now `List`s), and this line becomes

```scala
        val netCreated = (extraCreated ++ created).filterNot(c => consumed.contains(c.cid))
```

Is there still a performance issue I'm missing?